app/src/main/java/io/left/meshim/utilities/FileExtensions.java

```diff
@@ -0,0 +1,17 @@
+package io.left.meshim.utilities;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Generic file extensions
+ */
+
+public class FileExtensions {
+    public final static Set IMAGE = new HashSet<>(Arrays.asList("png", "jpg","svg"));
+    public final static String[] DOCUMENTS = new String[] {"txt","xlsx", "xls", "doc",
```

Comment:
Is there a reason we picked these and exclude others? Why not just allow everything?